context/context.go

```diff
@@ -194,6 +194,9 @@ type Context interface {
 
 	// ContextOptions returns the context's options.
 	ContextOptions() ContextOption
+
+	// GetInputNameForEvent return the inputName of the event
+	GetInputNameForEvent() string
```

Comment:
Could it be "GetInputName" or "GetEventInputName"?

Reply:
GetInputName is better, @zclonely would you change this accordingly?

Reply:
No problem. I will change it immediately!  (`GetInputNameForEvent` to `GetInputName`)
BTW, Is there any relevant documentation need update？

Reply:
No docs need to be updated so far